js/livecoin.js

```diff
@@ -563,6 +563,8 @@ module.exports = class livecoin extends Exchange {
                         throw new InvalidOrder (this.id + ': Unable to block funds ' + this.json (response));
                     } else if (error === 503) {
                         throw new ExchangeNotAvailable (this.id + ': Exchange is not available ' + this.json (response));
+                    }  else if (error === 429) {
```

Comment:
Hi)  Thx as always for your participation in this )

Quick question on this line: is this a HTTP code or their API error code? ;)

Reply:
I think API error code.

    livecoin {"success":false,"errorCode":429,"errorMessage":"Too Many Requests"} <class 'ccxt.base.errors.ExchangeError'>